iothub/device/src/ConnectionStatusChangeReason.cs

```diff
@@ -8,13 +8,59 @@ namespace Microsoft.Azure.Devices.Client
     /// </summary>
     public enum ConnectionStatusChangeReason
     {
+        /// <summary>
+        /// This signifies that the client is connected, and ready to be used.
```

Comment:
I don't think you need "this signifies". It can be similiar to the enum defined in ConnectionStatus.cs - /// <para>The client is connected, and ready to be used.</para>

Reply:
Same for all other enum values in this file.